### PR TITLE
Backend: Cleanup IslandExceptions to improve readability

### PR DESCRIPTION
**Pull request by walkerselby**
## What
Cleanup IslandExceptions to improve readability.

## Changelog Technical Details
+ Cleaned up IslandExceptions to improve readability. - walker

### CODE REVIEW

**Comment by Thunderblade73** [comment]
You can also use the IslandType.CATACOMBS instead of DungeonAPI.inDungeons.

**Comment by jani270** [comment]
YOOO walker prs 
image

**Comment by walkerselby** [comment]
> YOOO walker prs image

I am (slightly) back

File: src/main/java/at/hannibal2/skyhanni/data/mob/IslandExceptions.kt [hunk 1]
```diff
@@ -38,200 +37,215 @@ object IslandExceptions {
...
+        baseEntity is EntityOcelot && armorStand?.isDefaultValue() == false &&
+            armorStand.name.startsWith("§8[§7Lv155§8] §cAzrael§r") ->
+            MobUtils.getArmorStand(baseEntity, 1).makeMobResult { MobFactories.basic(baseEntity, it) }
+
+        baseEntity is EntityOcelot && (nextEntity is EntityOcelot || nextEntity == null) -> MobUtils.getArmorStand(
+            baseEntity, 3,
+        ).makeMobResult { MobFactories.basic(baseEntity, it) }
+
+        baseEntity is EntityOtherPlayerMP &&
+            baseEntity.name.let { it == "Minos Champion" || it == "Minos Inquisitor" || it == "Minotaur " } &&
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/data/mob/IslandExceptions.kt:201]: Make this a pattern.
```
Mino(?:s Champion|s Inquisitor|taur )
```

**Review by Thunderblade73 — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/data/mob/IslandExceptions.kt (hunk 1 above)
Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/data/mob/IslandExceptions.kt:201]: I dont like using a that complex regex here. rather use (x|y|z) for better readability

**Pull request merged**